django/thunderstore/repository/api/v1/viewsets.py

```diff
@@ -35,7 +36,12 @@ def get_no_cache_response(cls):
     def get_queryset(self):
         return get_package_listing_queryset(community_site=self.request.community_site)
 
-    @action(detail=True, methods=["post"], permission_classes=[IsAuthenticated])
+    @action(
+        detail=True,
+        methods=["post"],
+        authentication_classes=[SessionAuthentication, BasicAuthentication],
```

Comment:
This feels quite hacky but without it, `test_api_v1_rate_package_permission_denied` fails due to the different response (and it would be a breaking change to a stable API)

Reply:
So what you can do is use a global setting for authentication classes, but this is probably the way we want to go because we might not want to enable session authentication in future APIs. Session authentication enabled API endpoints are CSRF vulnerable and as such need CSRF checks

Reply:
The reason the change was needed was because of changing from the default to Token:
https://github.com/risk-of-thunder/Thunderstore/blob/bf7ed221630cb54898a37a57e7d2a50831fc3ed4/django/thunderstore/core/settings.py#L313-L315
The default is `[SessionAuthentication, BasicAuthentication]`

Reply:
Yeah, so it's exactly as it should be, nice 👍